### PR TITLE
Update enable-bitlocker.md

### PR DESCRIPTION
**Pull request by msbemba**
added an important Note

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by e0i** [comment]
@dariomws306 @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by dariomws306** [comment]
I think this note is not needed.

**Comment by tseward** [comment]
I agree with @dariomws306. Instead, this information should go into a general PSReadLine documentation as this will impact any inputted password when using `ConvertTo-SecureString`. Perhaps in the NOTE section of the doc at https://docs.microsoft.com/en-us/powershell/module/psreadline/about/about_psreadline and have `ConvertTo-SecureString` reference that note.

File: docset/windows/bitlocker/enable-bitlocker.md [hunk 1]
```diff
@@ -177,6 +177,10 @@ The command also specifies that this volume uses a combination of the TPM and th
 The command also specifies to encrypt the used space data on the disk, instead of the entire volume.
 When the system writes data to the volume in the future, that data is encrypted.
 
+> [!IMPORTANT]
+> if you enable BitLocker this way, the password will be stored in the following location as plain text:
```
Comment (JohanFreelancer9) [inline docset/windows/bitlocker/enable-bitlocker.md]:
```suggestion
> If you enable BitLocker in this way, the password will be stored in the following location as plain text:
```

**Comment by get-itips** [comment]
agree with @dariomws306 and @tseward

**Comment by msbemba** [comment]
> Should we apply this to 5.0, 5.1 and 7 versions?

The Change applies to all versions. I have created a new PR https://github.com/msbemba/PowerShell-Docs/pull/1 to reference converto-securestring

**Comment by dariomws306** [comment]
I see that @sdwheeler improve it and applied it only to 6 and 7 version. Does this not apply to version 5/5.1 too?

**Comment by e0i** [comment]
Closing as https://github.com/MicrosoftDocs/PowerShell-Docs/pull/4831 addresses the original issue. Thanks.

**Pull request closed without merging**